src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -882,7 +899,8 @@ Subgraph::SubgraphExecutor::SubgraphExecutor(const std::shared_ptr<Subgraph::Sub
                                              const std::vector<ptrdiff_t>& start_offset_in,
                                              const std::vector<ptrdiff_t>& start_offset_out,
                                              const std::shared_ptr<CPURuntimeConfig>& snippet_config,
-                                             const BufferScratchpadAllocator& allocator)
+                                             const BufferScratchpadAllocator& allocator,
+                                             const dnnl::engine& engine)
```

Comment:
Looks we use `engine` nowhere in executors. Can we remove this argument?

Reply:
Thanks for the catch, removed